frontend-instructor/src/hooks/useApi.jsx

```
@@ -1,13 +1,13 @@
-import useInstructorAuth from "@/src/context/authContext";
+import useInstructorAuth from '@/src/context/authContext';
 
 export default function useApi() {
   const { instructor } = useInstructorAuth();
 
   const fetchData = async (apiFunction, params) => {
-    console.log(apiFunction, params);
+    //console.log(apiFunction, params);
     let newParams = {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'newParams' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)